todo_app/app.py

```diff
@@ -12,24 +20,22 @@ def create_app():
     #when this file is imported
     app.config.from_object('todo_app.flask_config.Config')
     # All the routes and setup code etc
-
     @app.route('/')
     def index(): 
-        items = Get_items_from_trello()
+        items = Get_items_from_mongodb()
         item_view_model = ViewModel(items)
         return render_template("index.html", view_model=item_view_model)
         
     @app.route('/', methods=['POST'])
     def add_todo(): 
         item = request.form.get('additem')
-        create_card(item)
+        Create_iten_from_mongodb(item)
```

Comment:
Minor typo here:
```suggestion
Create_item_from_mongodb(item)
```

Regardless, the changes in this file are really nice and limited - this is a great example how you've kept the external interface virtually the same so that these functions don't need to care really what's storing the underling data, they work basically the same!